src/game/test/gameModal.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jest-environment jsdom
+ */
+"use strict";
+const { displayModal, closeModal } = require("../gameOverModal");
+
+describe("Use modal function", () => {
+  describe("called displayModal with provided parameters", () => {
```

Comment:
You can connect all cases under one and the same `describe`. After that just create a `test.each` for all three cases.